INSTALL.md

```diff
@@ -1801,7 +1801,7 @@ Test Failures
 If some tests fail, look at the output.  There may be reasons for the failure
 that isn't a problem in OpenSSL itself (like an OS malfunction or a Perl issue).
 
-You may want increased verbosity, that can be accomplished as described in
+You may want increased verbosity, which can be accomplished as described in
```

Comment:
How about two sentences?

`You may want increased verbosity. This can be accomplished...`